__tests__/app.test.js

```diff
@@ -436,3 +436,35 @@ describe("8. PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("9. GET /api/users", () => {
+  it("status:200, should responds with an array of topics", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users).toHaveLength(4);
+
+        users.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              username: expect.any(String),
+              name: expect.any(String),
+              avatar_url: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+
+  it("status:404, should responds with error message when the path is invalid", () => {
+    return request(app)
+      .get("/api/usersssssssss")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Route not found");
+      });
```

Comment:
Hey Ryan, no need to do any more bad path checks after this endpoint - once you've covered it once you know that your bad path handler is working